### ISSUE TITLE
enhancement Remote SSH

### ISSUE THREAD

**Issue by Abendsaufstehen** [SkEditorTeam/SkEditor#120]
lets you use any remote machine with a ssh server as your development environment
with automatic uploads and downloads files on change (sync) for better workflow

**Comment by NotroDev** [comment]
You can use software like WinSCP to edit remote files (you need to assign SkEditor for the chosen file types.

However, the integrated SFTP explorer will probably be added in the future.

**Comment by Abendsaufstehen** [comment]
Yep I know about WinSCP, i just thought a Integrated SFTP explorer would be great because VS Code also has a "Extension" called Remote SSH and it makes life just better by not making me drag all the files from my desktop on my server, its just Quality of Life tbh, but yea thanks tho

**Comment by NotroDev** [comment]
You don't need to move your files - WinSCP handles them. You can just open files from WinSCP, and when they're saved, WinSCP automatically uploads them to the server.

That's a pretty popular misconception, and I think I need to add some info somewhere.

btw, there's this tutorial from the old SkEditor+' wiki: https://github.com/NotroDev/SkEditorPlus/wiki/How-to-use-SkEditor-in-an-FTP-program%3F

**Comment by Justman100** [comment]
> However, the integrated SFTP explorer will probably be added in the future.

When?

**Comment by NotroDev** [comment]
> > However, the integrated SFTP explorer will probably be added in the future.
> 
> When?

I don't know. SkEditor is a project I'm developing in my free time, and I don't make any money from it. So when I work and what I work on depends on whether I can and want to do so.

**Comment by Justman100** [comment]
> > Because without SSH support, it's kind of useless!
> 
> ...have files accessed via the SFTP protocol.

And? SkEditor has no SFTP support!

**Comment by NotroDev** [comment]
> > > Because without SSH support, it's kind of useless!
> > 
> > ...have files accessed via the SFTP protocol.
> 
> And? SkEditor has no SFTP support!

Please read the entire conversation. WinSCP is a very solid and popular option for server file editing. A built-in SSH/SFTP explorer would be cool, but it would require tons of work for a small benefit. That's not something I want to work on right now.
